content/en/monitors/monitor_types/integration.md

```diff
@@ -11,22 +11,92 @@ further_reading:
   text: "Schedule a downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
-  text: "Consult your monitor status"
+  text: "Check your monitor status"
 ---
 
-{{< img src="monitors/monitor_types/integration/es_status.png" alt="es status" responsive="true" >}}
-
 ## Overview
 
-On the integration tab you see a list of your installed integrations. Upon selection, you can choose to monitor either a "Status" or a "Metric".
+Use an integration monitor to check if an installed integration is running. For more detailed monitoring, a metric monitor can be used to gauge specific information about an integration.
```

Comment:
```suggestion
Use an integration monitor to check if an installed [integration](/integrations/) is running. For more detailed monitoring, a [metric monitor](/monitors/monitor_types/metric/) can be used to gauge specific information about an integration.
```

Reply:
Added the link to integrations but adding the link to metric monitors here doesn't seem appropriate.